support/testsupport/src/main/java/com/hotels/styx/support/matchers/TransformingMatcher.java

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright (C) 2013-2018 Expedia Inc.
```

Comment:
Let's get rid of this class. You can just copy the changes from my closed PR: https://github.com/HotelsDotCom/styx/pull/478.